data-gathering/github_repos.json

```diff
@@ -39,6 +39,7 @@
 				{"name": "Puzzle ITC","orgs": ["puzzle"]},
 				{"name": "seantis gmbh","orgs": ["seantis"]},
 				{"name": "secanis","orgs": ["secanis"]},
+				{"name": "sourcepole","orgs": ["sourcepole"]},
```

Comment:
On line 44 we allready have sourcepole

Reply:
Ooops, I missed that one. I removed southpole from the PR.